src/hotspot/cpu/x86/interpreterRT_x86_32.cpp

```diff
@@ -44,6 +44,7 @@ InterpreterRuntime::SignatureHandlerGenerator::SignatureHandlerGenerator(const m
   _masm = new MacroAssembler(buffer);
 #ifdef AMD64
 #ifdef _WIN64
+   // FIXME: This is weird. How can we ever have _WIN64 for 32-bit code? I wonder what was meant. /ihse
```

Comment:
I think this piece of code will never get compiled and should be removed, and just the `#else` clause kept, but I guess some code archaeology is in place to figure out how and why this was added in the first place.

Reply:
I think this is a copy-paste error from [JDK-8199809](https://bugs.openjdk.org/browse/JDK-8199809): the code from `interpreterRT_x86_64.cpp` (where `WIN64` makes sense) was copy-pasted here in `interpreterRT_x86_32.cpp`. In fact, `AMD64` in  `interpreterRT_x86_64.cpp` makes no sense as well. I'll clean it up: [JDK-8343167](https://bugs.openjdk.org/browse/JDK-8343167).